test/multiple-user-principal-types.test.js

```diff
@@ -393,37 +395,194 @@ describe('Multiple users with custom principalType', function() {
           .then(function(isOwner) {
             expect(isOwner).to.be.true();
           });
-      });
-
-      it('expects OWNER to resolve false if owner has incorrect principalType', function() {
-        var Album = app.registry.createModel('Album', {
-          name: String,
-          userId: Number,
-        }, {
-          relations: {
-            user: {
-              type: 'belongsTo',
-              model: 'OneUser',
-              foreignKey: 'userId',
-            },
-          },
         });
-        app.model(Album, {dataSource: 'db'});
 
-        return Album.create({name: 'album', userId: userFromOneModel.id})
+        it('legacy behavior resolves false without belongsTo relation', function() {
```

Comment:
Makes sense, I think I was a bit confused and thought I was reviewing tests for the single-user scenario.

To prevent future confusion, could you please capture this important difference from the single-user scenario in a comment before the test?

```js
// With multiple users config, we cannot resolve a user based just on his id,
// as many users from different models could have the same id.
it('resolves false without belongsTo relation', function() {
// ...
```